tests/test_vrf.py

```diff
@@ -31,6 +31,11 @@
     So, we prefer a fixture rather than xunit-style setup/teardown functions.
 """
 
+@pytest.fixture(scope="module", autouse=True)
+def setup_check_topo(testbed):
+    if testbed['topo']['name'] != 't0':
+        pytest.skip('Unsupported topology')
```

Comment:
is this by design that vrf won't work for t0 topology?

Reply:
Run vrf on t0 only.